sgbatch/src/SimpleExecutionController.h

```diff
@@ -35,6 +35,10 @@ class SimpleExecutionController : public wrench::ExecutionController {
               //const double& hitrate,
               const std::string& outputdump_name);
 
+    std::map<std::string, JobSpecification>& get_workload_spec() {
```

Comment:
Is this getter method really needed? Couldn't this be accessed directly?

Reply:
I mean, that's the constant questions. Typically, keeping something private and adding a getter is the way to go. But since this getter returns a reference, as it should since we don't want the data copy, perhaps it's better to make the field public and remove the getter. I don't feel strongly one way or another. The "everything private and getters for everything" attitude is probably overkill.